site/content/docs/cli/constructs.md

```diff
@@ -0,0 +1,422 @@
+
+```js
+const { CheckGroup, ApiCheck } = require('@checkly/cli/constructs')
+
+const group = new CheckGroup('check-group-1', {
+  name: 'Group',
+  activated: true,
+  tags: ['api-group'],
+  concurrency: 10,
+  browserChecks: {
+    testMatch: '*.spec.js'
+  }
+})
+
+new ApiCheck('check-group-api-check-1', {
+  name: 'API check #1',
+  groupId: group.ref(),
+  request: {
+    method: 'GET',
+    url: 'https://mac-demo-repo.vercel.app/api/hello',
+  }
+})
+```
+
+- `name` : A human readable name for your Check Group.
+- `concurrency`: A number indicating the amount of concurrent checks to run when a group is triggered.
+- `locations`: An array of location codes where to run the checks in the group, i.e. `['us-east-1', 'eu-west-1]`.
+- `privateLocations`: An array of [Private Locations](https://www.checklyhq.com/docs/private-locations/) slugs, i.e. `['datacenter-east-1]`.
+- `alertChannels`: An array of `AlertChannel` objects to which to send alert notifications.
+- `activated`: A boolean value if all the checks in the group are activated.
+- `muted`: A boolean value if alert notifications from the checks in the group are muted, i.e. not sent out.
+- `tags`: An array of tags. Group tags trickle down to tags on the individual checks. i.e. `['product', 'api']`
+- `runtimeId`: The ID of which [runtime](https://www.checklyhq.com/docs/runtimes/specs/) to use for the checks in the group.
+- `environmentVariables`: An array of objects defining variables in the group scope, i.e. `[{ key: 'DEBUG', value: 'true' }]`
+- `localSetupScript`: Any JS/TS code as a string to run before each API check in this group.
+- `localTearDownScript`: Any JS/TS code as a string to run after each API check in this group.
+- `apiCheckDefaults`: A set of defaults for API checks. This should not be needed. Just compose shared defaults using JS/TS.
+- `browserCheckDefaults`: A set of defaults for API checks. This should not be needed. Just compose shared defaults using JS/TS.
+
+## Alert Channels
+
+Alert channels let you get alert notifications when a Check fails. [Learn more about alerting in our docs](https://www.checklyhq.com/docs/alerting/)
+All alert channels share a set of common properties to define when / how they should alert:
+
+- `sendRecovery`: A boolean if you want to receive recovery notifications.
+- `sendFailure`: A boolean if you want to receive failure notifications.
+- `sendDegrade`: A boolean if you want to receive degraded notifications. These only apply to API checks.
+- `sslExpiry`: A boolean if you want to receive a notification when a SSL/TLS certificate expires. This works only for API checks.
+- `sslExpiryThreshold`: A number indicating how many days before the certificate expiry date a notification will be triggered.
+
+Alert channels are assigned to Checks and CheckGroups by instantiating a class and adding the resulting object to the
+`alertChannels` array.
+
+### SMS Alert Channel
```

Comment:
Should also the specific alert channels link to their docs counterpart (e.g. `/https://www.checklyhq.com/docs/integrations/slack/`)?